spec/controllers/pageflow/files_controller_spec.rb

```diff
@@ -64,6 +64,37 @@ module Pageflow
           expect(response.status).to eq(404)
         end
       end
+
+      context 'https mode' do
+        let(:entry) { create(:entry, :published)}
+        let(:video_file) { create(:video_file, :entry => entry, :used_in => entry.published_revision)}
```

Comment:
Line is too long. [102/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space missing inside }.